test/jqLiteSpec.js

```diff
@@ -635,6 +635,56 @@ describe('jqLite', function() {
       expect(select.attr('multiple')).toBe('multiple');
     });
 
+    it('should set respective properties when getting/setting boolean attributes', function() {
```

Comment:
IMO it often does make sense to test things like that; in a way that we should have had a test like that in `1.5.x` and it should remain passing after we remove our own reflection. In this way we'd know if a buggy browser broke it and we need to patch it ourselves.

But maybe here we wouldn't do the patching either way so I'd be fine either way.